tests/integ/sagemaker/jumpstart/conftest.py

```diff
@@ -16,24 +16,44 @@
 import boto3
 import pytest
 from botocore.config import Config
+from sagemaker.jumpstart.hub.hub import Hub
 from sagemaker.session import Session
 from tests.integ.sagemaker.jumpstart.constants import (
     ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID,
+    ENV_VAR_JUMPSTART_SDK_TEST_HUB_NAME,
+    HUB_NAME_PREFIX,
     JUMPSTART_TAG,
+    SM_JUMPSTART_PUBLIC_HUB_NAME,
+)
+
+from sagemaker.jumpstart.types import (
+    HubContentType,
 )
 
 
 from tests.integ.sagemaker.jumpstart.utils import (
     get_test_artifact_bucket,
     get_test_suite_id,
+    get_sm_session,
 )
 
 from sagemaker.jumpstart.constants import JUMPSTART_DEFAULT_REGION_NAME
 
 
 def _setup():
     print("Setting up...")
-    os.environ.update({ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID: get_test_suite_id()})
+    test_suit_id = get_test_suite_id()
+    test_hub_name = f"{HUB_NAME_PREFIX}{test_suit_id}"
+    test_hub_description = "PySDK Integ Test Private Hub"
+
+    os.environ.update({ENV_VAR_JUMPSTART_SDK_TEST_SUITE_ID: test_suit_id})
+    os.environ.update({ENV_VAR_JUMPSTART_SDK_TEST_HUB_NAME: test_hub_name})
+
+    # Create a private hub to use for the test session
+    hub = Hub(
+        hub_name=os.environ[ENV_VAR_JUMPSTART_SDK_TEST_HUB_NAME], sagemaker_session=get_sm_session()
+    )
+    hub.create(description=test_hub_description)
```

Comment:
should we necessarily create a Hub _every_ time a JS integ test is run? Does this bring any problems?

Reply:
I don't think of any problems tbh with this strategy, we're cleaning it up in the end. Why do you think we should be approaching it differently?